templates/events-list.php

```diff
@@ -23,9 +35,12 @@
 		<?php
 		while ( $query->have_posts() ) :
 			$query->the_post();
+			$event_start = ( new DateTime( get_post_meta( get_the_ID(), '_event_start', true ) ) )->format('l, F j, Y');
 			?>
-			<li>
+			<li class="event-list-item">
+				<span class="event-list-date"><?php echo esc_html( $event_start ); ?></span>
 				<a href="<?php the_permalink(); ?>"><?php the_title(); ?></a> by <span><?php the_author(); ?></span>
```

Comment:
This change will get the correct link to the template:
- Example of a previous link: https://glotpress.test/events/upcoming-event/
- Example of a new link: https://glotpress.test/glotpress/events/upcoming-event/ 

You need to add previously this code: 

```
$post_slug = get_post_field( 'post_name', get_the_ID() );
```

```suggestion
				<a href="<?php echo esc_url( gp_url( '/events/' . $post_slug ) ) ?>"><?php the_title(); ?></a> by <span><?php the_author(); ?></span>
```

Reply:
As I already mentioned elsewhere, I think it'd be ok to take a look at this later. But if we need to do it, we should continue to use the permalink function like this:
```suggestion
				<a href="<?php echo esc_url( gp_url( get_the_permalink() ) ); ?>"><?php the_title(); ?></a> by <span><?php the_author(); ?></span>
```

Reply:
I think we can fix this in a follow-up PR after we agree on the best approach.